packages/gatsby-theme/src/data.json

```diff
@@ -0,0 +1,22 @@
+{
+  "nodes": [
+    {
+      "name": "underConstruction",
+      "featuredImage": {
+        "node": {
+          "localFile": {
+            "childImageSharp": {
+              "fluid": {
+                "aspectRatio": 1,
+                "base64": "data:image/jpeg;base64,/9j/2wBDAAgGBgcGBQgHBwcJCQgKDBQNDAsLDBkSEw8UHRofHh0aHBwgJC4nICIsIxwcKDcpLDAxNDQ0Hyc5PTgyPC4zNDL/2wBDAQkJCQwLDBgNDRgyIRwhMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjL/wgARCAALABQDASIAAhEBAxEB/8QAFwAAAwEAAAAAAAAAAAAAAAAAAAQGAv/EABYBAQEBAAAAAAAAAAAAAAAAAAIBA//aAAwDAQACEAMQAAABZ1ZJKQxQmg//xAAbEAEAAgIDAAAAAAAAAAAAAAABAAMCBBESIv/aAAgBAQABBQKrT926uBT0iDLsSIc//8QAFhEBAQEAAAAAAAAAAAAAAAAAABEB/9oACAEDAQE/AZiP/8QAFxEBAQEBAAAAAAAAAAAAAAAAAQAREv/aAAgBAgEBPwHVbq//xAAeEAAABAcAAAAAAAAAAAAAAAAAARAxAhESISIjUf/aAAgBAQAGPwLYWIqhefUuQZP/xAAaEAEAAwEBAQAAAAAAAAAAAAABABEhMUHB/9oACAEBAAE/IULYRuJh5ht+S7pKahqJOMnmE//aAAwDAQACAAMAAAAQMN//xAAWEQEBAQAAAAAAAAAAAAAAAAABERD/2gAIAQMBAT8QAJM//8QAFhEBAQEAAAAAAAAAAAAAAAAAEQAh/9oACAECAQE/ENxJX//EABsQAQEBAQEAAwAAAAAAAAAAAAERIQAxUZHx/9oACAEBAAE/EKD0qpshpvqcnpz2DaRHycGAUfGcrplKWPbFNSnl/OOw+rv/2Q==",
+                "sizes": "(max-width: 800px) 100vw, 800px",
+                "src": "/static/d42542b913d6a0df2f720e43556b3a8b/f2cbb/underConstruction.jpg",
+                "srcSet": "/static/d42542b913d6a0df2f720e43556b3a8b/8850c/underConstruction.jpg 200w,\n/static/d42542b913d6a0df2f720e43556b3a8b/da113/underConstruction.jpg 400w,\n/static/d42542b913d6a0df2f720e43556b3a8b/f2cbb/underConstruction.jpg 800w,\n/static/d42542b913d6a0df2f720e43556b3a8b/1789c/underConstruction.jpg 960w"
```

Comment:
To prevent errors, I don't know if it's a good idea to make these empty strings, in order to just keep the base64 version of the image